fdbserver/Ratekeeper.actor.cpp

```diff
@@ -291,7 +297,13 @@ void updateRate( Ratekeeper* self ) {
 		}
 
 		int64_t storageQueue = ss.lastReply.bytesInput - ss.smoothDurableBytes.smoothTotal();
+		self->healthMetrics.storageStats[ss.id].storageQueue = storageQueue;
```

Comment:
very minor, but consider doing a single map lookup, and then setting all four values:

auto& ssMetrics = self->healthMetrics.storageStats[ss.id];
ssMetrics.storageQueue = storageQueue;
ssMetrics.storageDurabilityLag = storageDurabilityLag;
ssMetrics.cpuUsage = ss.lastReply.cpuUsage;
ssMetrics.diskUsage = ss.lastReply.diskUsage;